src/GitHub.App/Services/RepositoryPublishService.cs

```diff
@@ -38,11 +39,7 @@ public string LocalRepositoryName
             IApiClient apiClient)
         {
             return Observable.Defer(() => apiClient.CreateRepository(newRepository, account.Login, account.IsUser)
-                         .Select(remoteRepo => new { RemoteRepo = remoteRepo, LocalRepo = activeRepository }))
-                    .SelectMany(repo => gitClient.SetRemote(repo.LocalRepo, "origin", new Uri(repo.RemoteRepo.CloneUrl)).Select(_ => repo))
-                    .SelectMany(repo => gitClient.Push(repo.LocalRepo, "master", "origin").Select(_ => repo))
-                    .SelectMany(repo => gitClient.Fetch(repo.LocalRepo, "origin").Select(_ => repo))
-                    .SelectMany(repo => gitClient.SetTrackingBranch(repo.LocalRepo, "master", "origin").Select(_ => repo.RemoteRepo));
+                                     .Select(remoteRepo => new { RemoteRepo = remoteRepo, LocalRepo = activeRepository }))
```

Comment:
Did you miss something from the commit? This is missing the setup and doesn't even look like it will compile...